### ISSUE TITLE
Conda Package?

### ISSUE THREAD

**Issue by Jlaventall** [elimintz/justpy#156]
Would it be possible to have justpy hosted as a conda package so that it could be managed in conda environment?

**Comment by elimintz** [comment]
Do any of the options described in the link below work for you?
https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-pkgs.html#installing-non-conda-packages

**Comment by Jlaventall** [comment]
Yes. I was able to create a Conda virtual env and install it via pip inside the virtual env. Since I use Conda as my package manager, it would be ideal to have justpy available as a Conda package directly.

**Comment by elimintz** [comment]
I am not that familiar with Conda. How would I go about doing it? Is this a decision Anaconda makes?

**Comment by Jlaventall** [comment]
It appears they have a submission/review process:
[(https://conda-forge.org/docs/maintainer/adding_pkgs.html#)]

Putting it up on Conda Forge:
[(https://conda-forge.org/docs/user/introduction.html#what-is-conda-forge)]
would place it on the primary public channel, so that installation from a users perspective would be as simple as: 'conda install justpy' in the Anaconda CLI. Conda takes care of the rest of the installation and the user has the package at their disposal for use in virtual environments.

I bring this up as a request, because I believe there are many python users who use Conda as their package manager, particularly on the data science side. It saves a lot of time over using pip. While I have a working setup with pip, it doesn't integrate as well as it would if it was managed via Conda.
Thanks.

**Comment by matthewhoendorf** [comment]
Saving your dependencies in an ```environment.yml``` might be the best temporary solution. See https://stackoverflow.com/a/32799944.

Personally, I've switched from Conda to Poetry for environment and package management. It has made web app deployment a lot easier.

**Comment by Jlaventall** [comment]
I agree, the environment.yml method appears to be a sufficient solution for the meantime. Official docs if anyone needs it:
[https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file](url)

I did not know about Poetry. Thanks.
